### DISCUSSION TITLE
Cluster Config help

### DISCUSSION THREAD

**Discussion by thapld** [OvenMediaLabs/OvenMediaEngine#1199] [Help] [answered]
Sorry, I reopen and update question
I have 2 servers:
origin: 34.87.2.242
edge: 35.187.239.255

No firewall server at all

Server.xml and Log in origin server:
[https://gist.github.com/thapld/4ffee6b2a5df93cf1fec95c3d6d68163](url)

Server.xml and Log in edge server:
[https://gist.github.com/thapld/ae2ce1d9c4c2ca075e92ce41b5b59121](url)

1. Publish stream with obs
![image](https://user-images.githubusercontent.com/38736191/234502865-e4210bf9-5eb9-4882-a6b5-a43fc5dc1855.png)

3. Player
![image](https://user-images.githubusercontent.com/38736191/234503003-739bbe12-b8ad-457a-ab17-4216a3549fd7.png)

But I cannot play stream from edge server:
![image](https://user-images.githubusercontent.com/38736191/234503166-ed96a6b4-d7d9-4237-8b19-4eb29c972268.png)

Where I wrong, please help me. Thank you so much.

**Comment by getroot** [comment]
Please upload your ovenmediaengine.log files of Origin and Edge

**Comment by thapld** [comment]
I did update log

**Comment by getroot** [comment]
Perhaps you uploaded the wrong files.

First of all, you said you played ws://ip:3333/app/stream01, but there is no application named app in your Origin Server.xml. Your application name is `<Application> <Name>stream01</Name>`. This probably shouldn't work. The same goes for Edge's Server.xml.

And in your Origin's Server.xml there is a `<SessionLoadBalancingThreadCount>` setting, which is a long deprecated option. So the log file you provided and Server.xml do not match.

Your Edge's ovenmediaengine.log file shows no signs of the player trying to connect. This is also the weird part. Did you modify Logger.xml?

**Comment by thapld** [comment]
I reopen and update question.

**Comment by getroot** [comment, marked as answer]
Your edge's webrtc signaling port is 3334.

Below is your Server.xml of Edge
```
<Signalling>
				<Port>${env:OME_SIGNALLING_PORT:3334}</Port>
```

So you have to connect to ws://edge_ip:3334/stream01/test

**Comment by thapld** [comment]
OMG!. Thank you so much.
